HarfangLab/harfanglab/ingest/parser.yml

```
@@ -188,13 +188,78 @@ stages:
           event.dataset: "{{json_event.message.log_type}}"
           user.roles: "{{ json_event.message.groups | map(attribute='name') | join(',') | replace('\n', '') | replace(' ', '') }}"
           organization.id: "{{json_event.message.tenant}}"
-          url.original: "{{json_event.message.details_url_request.url}}"
+          url.original: "{{json_event.message.get('source_url') or json_event.message.details_url_request.url}}"
+          process.executable: "{{json_event.message.image_name or json_event.message.source_image or json_event.message.process_image_path}}"
+          process.pid: "{{json_event.message.pid}}"
+          process.command_line: "{{json_event.message.process_commandline}}"
+          action.properties.StackTrace: "{{json_event.message.stacktrace}}"
+          action.properties.EventId: "{{json_event.message.event_id}}"
+          action.properties.DesiredAccess: "{{json_event.message.desired_access_id}}"
+          action.properties.DesiredAccessStr: "{{json_event.message.desired_access_str}}"
+          action.properties.DeviceName: "{{json_event.message.device_name}}"
+          action.properties.SockType: "{{json_event.message.sock_type}}"
+          action.properties.Family: "{{json_event.message.family}}"
+          action.properties.TamperFlag: "{{json_event.message.tamper_flag}}"
+          action.properties.TamperFlagStr: "{{json_event.message.tamper_flag_as_str}}"
+          action.properties.ProcessEntrypointMemory: "{{json_event.message.process_entrypoint_memory}}"
+          action.properties.ProcessHeaderMemory: "{{json_event.message.process_header_memory}}"
+          action.properties.Access: "{{json_event.message.access}}"
+          action.properties.AccessStr: "{{json_event.message.access_str}}"
+          action.properties.IpAddresses: "{{json_event.message.ip_addresses}}"
```

Isn't `host.ip` more suitable?
```suggestion
          host.ip: "{{json_event.message.ip_addresses}}"
```